### ISSUE TITLE
Send "training loss" from Clients to Server 

### ISSUE THREAD

**Issue by minseok-ryu** [APPFL/APPFL#61]
"Training loss" computed from clients can be used to track how well the FL is running and for a hyperparameter tuning process.
However, there may be the possible data leakage from communicating "training loss" from clients to server.

**Comment by youngdae** [comment]
Would the use of `secure_channel` of gRPC requiring SSL/TLS help resolve this potential issue?
